state-network.md

```diff
@@ -145,6 +155,7 @@ A leaf node from a contract storage trie and accompanying merkle proof against t
 storage_trie_proof_key := Container(address: Bytes20, slot: uint256, state_root: Bytes32)
 selector               := 0x01
 
+content                := Container(witnesses: mpt_witnesses, data: Bytes32)
```

Comment:
What is data here?

Reply:
Just the value being stored in the slot.

Reply:
maybe lets name it `slot_value`?

Though, I'm curious whether we should just use the `witnesses` part for this since the witness should contain the data...

Reply:
You may be right.  I didn't deeply inspect the contents of the witnesses we were generating locally since my working code doesn't do proof validation yet when receiving content.  If that is the case, presumably it would be in the last node of the witnesses, right?  I can dig into it and see what additions should be made.